src/utils.ts

```diff
@@ -22,6 +22,10 @@ export function slash(p: string): string {
   return p.replace(windowsSlashRE, '/')
 }
 
+export function resolveConfigDir(path: string, root = process.cwd()) {
+  return path.replace('${configDir}', root)
+}
+
```

Comment:
I think the `root` is better to be required.

Reply:
And I'm not sure whether the `${configDir}` placeholder is force at the beginning.

Reply:
I made it required. 
I don't see a case where ${configDir} would not be at the start. All the example in the typescript documentation are located at the start of the string